Libraries/LibJS/Console.h

```diff
@@ -110,6 +112,8 @@ class ConsoleClient : public Cell {
     GC_DECLARE_ALLOCATOR(ConsoleClient);
 
 public:
+    virtual u64 tag() const override { VERIFY_NOT_REACHED(); }
```

Comment:
This should probably just be made a `GC::Cell` instead of `JS::Cell` to avoid this type of problem